devito/ir/clusters/algorithms.py

```diff
@@ -388,7 +388,11 @@ def guard(clusters):
                 guards = {}
                 for d in e.conditionals:
                     condition = guards.setdefault(d.parent, [])
-                    condition.append(d.condition or CondEq(d.parent % d.factor, 0))
+                    
```

Comment:
nitpicking: unnecessary blank line -- I don't think the line `condition = ...` should be logically separated from the if-then below